src/sshagent/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 if(WITH_XC_SSHAGENT)
+    include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
How about moving this line out of ```if(WITH_XC_SSHAGENT)```? This fix is for a generic issue regarding Qt and CMake, not sshagent-specific. Maybe in the future some other parts in KeePassXC will need this trick, too.

Reply:
This line will include files from the current directory, `src/sshagent`. 

If we need it in other parts we need to add a CMakeFile.txt in that specific folder and add this line.
`src/http` directory is an example as well.
Those 2 folders are loaded in `src/CMakeFile.txt`, but here files were not included.